bleak/backends/corebluetooth/client.py

```diff
@@ -354,14 +354,19 @@ async def start_notify(
         self,
         characteristic: BleakGATTCharacteristic,
         callback: NotifyCallback,
+        cb: Optional[Dict[str, Any]] = None,
```

Comment:
```suggestion
        *,
        cb: Dict[str, Any],
```

This will make it a keyword only arg. (we're doing the same elsewhere already)

Reply:
Made a keyword only arg with default of {}